<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelTimePattern</class>
 <widget class="QDialog" name="PanelTimePattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time pattern</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_path">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>10</y>
     <width>475</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select a configuration file</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Filename:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_cmds">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>621</width>
     <height>401</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time / ms</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Filename</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Spectra</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Integration time / ms</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Averaging</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_current">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Current:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_current">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>45</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>40</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_total">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>45</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_device">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_device">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
